entry

```diff
@@ -10,53 +86,15 @@ update-ca-certificates
 tiller --listen=127.0.0.1:44134 --storage=secret &
 export HELM_HOST=127.0.0.1:44134
 
-helm init --skip-refresh --client-only
-helm repo update --strict || helm repo remove stable
-if [ -n "$REPO" ]; then
-    helm repo add ${NAME%%/*} $REPO
+helm_v2 init --skip-refresh --client-only
+EXIST=$(helm_v2 ls --all "^$NAME\$" --output json | jq -r '.Releases | length')
```

Comment:
How much overhead or time does it take to start tiller each time and run `helm ls`?  Does it do anything like create a configmap in k8s?

Reply:
its really quick process, it starts a daemon that connects to kubernetes to read releases from secrets: 
tiller --listen=127.0.0.1:44134 --storage=secret

and helm init afterwards just initialize the .helm directory within the pod which also a quick process